docs/configuration-files/add-configuration-file.md

```diff
@@ -12,3 +12,12 @@ links:
     related:
     featured:
 ---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Applications` card
+1. Click `View` in the `Applications` card to see a list of existing `Applications`
+4. In the list of applications find the application where you want to add a configuration file and click on`DETAILS`
```

Comment:
There is a space missing after `on`. In other topics we usually favor just saying `click <thing to be clicked>`. The more straightforward the instructions are less friction we're gonna have correcting English grammar on our docs.

Reply:
I agree, just removed the 'on'.